contextMenu.js

```diff
@@ -135,6 +136,26 @@
                             }
                         }
 
+						//update ico
+                        if (ico) {
+                            var imgIcon = elm.find('.iw-mIcon');
+                            if (imgIcon.length) {
+								if(imgIcon[0].tagName.toLowerCase() === "span"){
+									var classList = ["iw-mIcon"];
+									if(typeof ico == "string")
+										classList.push(ico);
+									else
+										classList.push.apply(classList, ico);
+									imgIcon[0].classList = classList
+								} else {
+									imgIcon[0].remove()
+									elm.prepend('<span align="absmiddle" class="iw-mIcon '+ico.toString().replace(",", " ")+'" />');
+								}
+                            } else {
+                                elm.prepend('<span align="absmiddle" class="iw-mIcon '+ico.toString().replace(",", " ")+'" />');
+                            }
+                        }
+						
```

Comment:
I feel it will be better just remove iw-mIcon and prepend it again based on ico/img. Something like this.
```js
if(imgIcon){ imgIcon.remove(); }
if (img) {
      elm.prepend('<img src="' + img + '" align="absmiddle" class="iw-mIcon" />');
} else if (ico) {
      elm.prepend('<span align="absmiddle" class="' + ("iw-mIcon "+ico+'" />');
 }
```